doc/source/theory/FlavorSpace.rst

```diff
@@ -236,7 +236,7 @@ Operator Anomalous Dimension Basis
 Operator Intrinsic QCD Evolution Basis
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- here we mean :ref:`theory/FlavorSpace:Intrinsic QCD Evolution Bases` both in the input and the output space
+- here we mean :ref:`theory/FlavorSpace:Intrinsic |QCD| Evolution Bases` both in the input and the output space
```

Comment:
Pay attention, reStructuredText does not support nesting in general, so don't use markup inside markup, unless you explicitly checked that is working (so remove pipes `|` around `QCD` inside the backticks ``` ` ```).

Reply:
You are referencing the heading here and that name is without the abbreviation - which is correct